### PR TITLE
MonthlySMS and SMS

### PR DESCRIPTION
**Pull request by benrudolph**
@czue @nickpell this is admittedly ugly, but couldn't see a better way to do this.
http://manage.dimagi.com/default.asp?183551#1024474

cc: @TylerSheffels

### CODE REVIEW

File: corehq/apps/accounting/static/accounting/ko/accounting.credits.js [hunk 1]
```diff
@@ -67,4 +68,11 @@ var CreditItem = function (category, data, paymentHandler, can_purchase_credits)
             creditItem: self
         }));
     };
+
+    /*
+     * Return the name with the recurring interval if it exists
```
Comment (benrudolph) [inline corehq/apps/accounting/static/accounting/ko/accounting.credits.js]: maybe frivolous?

File: corehq/apps/accounting/static/accounting/ko/accounting.credits.js [hunk 2]
```diff
@@ -67,4 +68,11 @@ var CreditItem = function (category, data, paymentHandler, can_purchase_credits)
...
+     */
+    self.getUsageName = function() {
+        return self.recurringInterval() ? self.recurringInterval() + ' ' + self.name() : self.name()
```
Comment (dimagimon) [inline corehq/apps/accounting/static/accounting/ko/accounting.credits.js]: Missing semicolon.

File: corehq/apps/accounting/static/accounting/ko/accounting.credits.js (hunk 1 above)
Comment (czue) [inline corehq/apps/accounting/static/accounting/ko/accounting.credits.js]: /shrug

**Comment by czue** [comment]
:+1: seems as good as anything

**Pull request merged**